src-java/northbound-service/northbound/src/test/java/org/openkilda/northbound/controller/v2/YFlowControllerV2Test.java

```diff
@@ -95,9 +116,92 @@ public void shouldReturnsStatus400WhenPathComputationStrategyNull() throws Excep
                 .andExpect(status().isBadRequest());
     }
 
+    @Test
+    @WithMockUser(username = TestConfig.USERNAME, password = TestConfig.PASSWORD, roles = TestConfig.ROLE)
+    public void shouldReturnStatus400WhenSubFlowVlanIdIsInvalid() throws Exception {
+        verifyCreateRequestFailsOnBadVlanId(4096, 0, 0, 0);
```

Comment:
Please rewrite to @ParametrizedTest. You always should be guided by the principle 'one test - one assertion'.